subprojects/distributions/binary-compatibility.gradle

```diff
@@ -97,6 +99,19 @@ projects.each { projectName ->
     }
 }
 
+configurations {
+    currentClasspath {
```

Comment:
why not just `extendsFrom` classpaths in the other projects?

Reply:
I think the `Usage` approach is the right one.  The configuration in another project just happens to be named `runtimeClasspath`, but we really want the "runtime variant".